metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -338,8 +340,8 @@ def get_dataflow_graph(self, script_path: str) -> Optional[Dict[str, Any]]:
         try:
             obj = self.s3_client.get_object(Bucket=bucket, Key=key)
         except botocore.exceptions.ClientError as e:
-            self.report.report_failure(
-                script_path,
+            self.report_warning(
+                flow_urn,
                 f"Unable to download DAG for Glue job from {script_path}, so job subtasks and lineage will be missing: {e}",
```

Comment:
If the glue script can not be accessed due to some reason (missing permission - 403 Or missing script - 404), treat this as warning than error.